bower.json

```diff
@@ -40,7 +40,7 @@
     "marked-element": "PolymerElements/marked-element#^1.2.0",
     "prism-element": "PolymerElements/prism-highlighter#^1.2.0",
     "lazy-imports": "Polymer/lazy-imports#^0.0.2",
-    "web-components": "git@github.com:KanoComputing/web-components.git",
+    "web-components": "git@github.com:KanoComputing/web-components.git#master",
```

Comment:
Could we remove the branch flag and reset this to using the released version? That gives us a bit more control over releasing features, rather than having the live site pull in the latest developments.